networking_dvs/utils/dvs_util.py

```diff
@@ -445,16 +436,35 @@ def create_dvportgroup(self, sg_attr_key, sg_set, port_config):
         except vmware_exceptions.VimException as e:
             raise exceptions.wrap_wmvare_vim_exception(e)
 
+    def dvportgroup_name(self, sg_set):
+        """
+        Returns a dvportgroup name for the particular security group set
+        in the context of the current switch
+        """
+        # There is an upper limit on managed object names in vCenter
+        dvs_id = ''.join(self.uuid.split(' '))[:8]
+        name = dvs_id + "-" + sg_set
```

Comment:
Could you change it back to 

`name = sg_set - "-" + dvs_id`

after all? I was first thinking, it was better to order it first by switch, then by security group, but then you can do that by simply going on the switch and listing the port-groups there.